app/views/logs/_log_filters.html.erb

```diff
@@ -83,7 +93,7 @@
           } %>
         <% end %>
 
-        <% if (current_user.support? || current_user.organisation.managing_agents.count > 1) && request.path == "/lettings-logs" %>
+        <% if (current_user.support? || non_support_with_multiple_managing_orgs?) && user_or_org_lettings_path? %>
```

Comment:
have refactored this to clarify, and fixed a bug in the process (in the old diff this was not showing for support users looking at a particular org